### PR TITLE
Lambda runtime update and add security headers

### PR DESCRIPTION
**Pull request by MichaelKuhinica**
Start by updating Node.js to latest version available on lambda@edge, which is 10.x, and making sure update worked as expected.
Then add the `Strict-Transport-Security` headers as per [this ticket](https://vimaly.com/#j8mqm/t/implement_Strict-Transport-Security_headers./8QoB8eMTz4xpHI7A0).

### CODE REVIEW

File: lambda/README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
   - Run `yarn jest`.
 
 ## Deployment
-  - Always run these steps on the `develop` branch.  There could be multiple testing versions of this lambda file around and you should always work with the merged copy, not your own copy, when modifying staging.
+  - Merge your changes to the `develop` branch before running this steps. There could be multiple testing versions of this lambda file around and you should always work with the merged copy, not your own copy, when modifying staging.
```
Comment (kylorhall) [inline lambda/README.md]:
```suggestion
  - Merge your changes to the `develop` branch before running these steps. There could be multiple testing versions of this lambda file around and you should always work with the merged copy, not your own copy, when modifying staging.
```

File: lambda/localizationLambda.js [hunk 2]
```diff
@@ -10,7 +10,7 @@
  * If you are viewing this file inside of AWS, this code comes from https://github.com/sharesight/www.sharesight.com/.
  */
 
-const version = '5';
+const version = '8';
```
Comment (kylorhall) [inline lambda/localizationLambda.js]: We were on 5 even?  Weird..

Comment (MichaelKuhinica) [reply lambda/localizationLambda.js]: I don't think there is an easy way to get this value from the context or update it automatically.

**Comment by kylorhall** [comment]
@MichaelKuhinica Did we release this?  It's still an open PR.

**Comment by MichaelKuhinica** [comment]
@kylorhall it's still an open PR, do we did not release it. [Here](https://vimaly.com/#j8mqm/t/AWS_Lambda_Node_v8_is_EOL_migrate_to_Node_v10_by_J.../8BuKRRg9U3FTf6fhk) is the ticket that was moved into ship it today. Is there any problem with it?

**Comment by kylorhall** [comment]
@MichaelKuhinica Nope, I just thought we deployed this a while ago is all and forgot to merge it across (given the "deploy" steps are manual and the code is just "storage").

No worries.

**Pull request merged**